morebits.js

```diff
@@ -569,7 +573,18 @@ Morebits.quickForm.element.prototype.compute = function QuickFormElementCompute(
 				subnode.setAttribute('value', data.value);
 			}
 			subnode.setAttribute('name', data.name);
-			subnode.setAttribute('type', 'text');
+
+			if (data.type === 'input') {
+				subnode.setAttribute('type', 'text');
+			} else {
+				subnode.setAttribute('type', 'number');
+				['min', 'max', 'step', 'placeholder', 'list'].forEach(function(att) {
```

Comment:
Not strictly related to this PR, but placeholder is valid (and quite useful) for input type=text too so can be added there.

Reply:
Fair point, although I think in general it's to be avoided (though I personally like 'em). Might as well now.  Done, with some other reorganizing.